### DISCUSSION TITLE
Track Browse Items Pagination in Browser History

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3100] [Feature Requests]
Right now, if you click through pages on Browse, and then click "back" in your browser, it will take you back to your pre-pagination page (eg: from Browse to Files), but a user might expect to go back a Browse results page (eg: from 6 to 5).

**Comment by benhaynes** [comment]
**Rijk**: We gotta figure out what we want this to do ideally. From what it sounds like, we’d want to use a query param of sorts for paging? The downside is that it’s based on layout so won’t always be supported 🤔

**Ben**: Yeah, I think query parameter is best... since the route could be used for deeply nested groups/folders. I think it’s important enough to include in each layout (where is makes sense).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
